src/analytics/lambdas/refresh-materialized-views-workflow/check-next-refresh-view.ts

```diff
@@ -18,6 +18,7 @@ export interface RefreshViewOrSp {
   name: string;
   type: string;
   timezoneSensitive: string;
+  viewName?: string;
```

Comment:
Is it the same as the `name`?

Reply:
the name is sp name, viewName is the table name that is created after sp refreshed
for example:
name: clickstream_acquisition_country_new_user_sp
viewName: clickstream_acquisition_country_new_user